src/templates/DatasetSearch/DatasetSearch.tsx

```diff
@@ -9,42 +9,50 @@ import LargeFileInfo from '../../components/LargeFileInfo';
 import SearchButton from '../../components/SearchButton';
 import PageHeader from '../../components/PageHeader';
 import { useQuery } from '@tanstack/react-query';
-import { separateFacets, updateSelectedFacetObject, selectedFacetsMessage, transformUrlParamsToSearchObject } from '../../services/useSearchAPI/helpers';
+import { separateFacets, transformUrlParamsToSearchObject } from '../../services/useSearchAPI/helpers';
 
 import axios from 'axios';
 import './dataset-search.scss';
+import { DatasetSearchPageProps, SelectedFacetsType, SidebarFacetTypes } from '../../types/search';
+import { TextFieldValue } from '@cmsgov/design-system/dist/types/TextField/TextField';
+
+const DatasetSearch = (props: DatasetSearchPageProps) => {
+  const {
+    rootUrl,
+    surveyLink,
+    additionalParams,
+    enableSort,
+    pageTitle,
+    filterTitle,
+    showSort,
+    showLargeFileWarning,
+    introText
+  } = props;
+
+  const defaultSort = { defaultSort: 'modified', defaultOrder: 'desc' }
+  const sortOptions = [
```

Comment:
Will this override the defaults if they need to be changed? Open Payments uses temporal fields to set their sort options instead of the normal title or modified date.